src/CVSelect/ContentViewList.js

```diff
@@ -65,8 +65,11 @@ export default function ContentViewList() {
 
     contentViews.forEach((cv) => {
       const str = cv.name.toLowerCase();
```

Comment:
If we're checking this in lowercase, we should also make sure debouncedsearchterm is lowercase.